camp/camp-brooklyn/src/test/java/org/apache/brooklyn/camp/brooklyn/ApplicationsYamlTest.java

```diff
@@ -232,6 +232,25 @@ public void testDoesNotWrapCatalogItemAndUsesCatalogServiceName() throws Excepti
         assertDoesNotWrap(app, BasicApplication.class, "catalogServiceLevel");
     }
 
+    @Test
+    public void testUnwrappedChildNotTagged() throws Exception {
+        addCatalogItems(
+                "brooklyn.catalog:",
+                "  id: simple",
+                "  version: " + TEST_VERSION,
+                "  item:",
+                "    type: " + BasicEntity.class.getName());
+        Entity app = createAndStartApplication(
+                "services:",
+                "- type: simple:" + TEST_VERSION);
+        Entity entity = Iterables.getOnlyElement(app.getChildren());
+        assertNull(entity.getConfig(EntityManagementUtils.WRAPPER_APP_MARKER));
+        // Note that "brooklyn.wrapper_app" will still make it into 
+        //   ((EntityInternal) entity).config().getBag().getAllConfigAsConfigKeyMap();
+        // so the UI will still show the marker as inherited by the entity.
```

Comment:
It's annoying that the UI will still show it. I guess that's because `getBag()` is also asking the parent for its bag? I'd have thought `.inheritance(ConfigInheritance.NONE)` would be enough to prevent that, but sounds like I would be wrong!